### PR TITLE
picom-allusive: 0.3.2 -> 1.2.3

### PR DESCRIPTION
**Pull request by allusive-dev**
## Description of changes

picom-allusive: 0.3.2 -> 1.2.3

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/window-managers/picom/picom-allusive.nix [hunk 1]
```diff
@@ -19,7 +19,7 @@ picom.overrideAttrs (oldAttrs: rec {
   '' + (lib.optionalString (oldAttrs ? postInstall) oldAttrs.postInstall);
 
   meta = (builtins.removeAttrs oldAttrs.meta [ "longDescription" ]) // {
-    description = "A fork of picom featuring improved animations and other features";
+    description = "The only Picom fork you will ever need. With Animations and More.";
```
Comment (eclairevoyant) [inline pkgs/applications/window-managers/picom/picom-allusive.nix:22]: desc shouldn't end in a period
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

**Review by eclairevoyant — CHANGES_REQUESTED** [review]
Also, please update pr/commit message to follow the format mentioned in contributing.md

**Review by IogaMaster — APPROVED** [review]
Result of `nixpkgs-review pr 264565` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>picom-allusive</li>
  </ul>
</details>

**Review by cafkafk — CHANGES_REQUESTED** [review]
Your commit summary should be `picom-allusive: 0.3.2 -> 1.2.2`

**Review by eclairevoyant — APPROVED** [review]

**Comment by allusive-dev** [comment]
I'm closing this PR as a update will release soon superseding this.

**Comment by IogaMaster** [comment]
To clarify allusive and I added a flake and also discovered that the chmod is not required. So I will make the aforementioned pr.

**Comment by IogaMaster** [comment]
https://github.com/NixOS/nixpkgs/pull/265109
@eclairevoyant

**Pull request closed without merging**